std/internal/math/gammafunction.d

```diff
@@ -1571,3 +1571,64 @@ unittest {
     for(auto x = 1.0; x < 15.0; x += 1.0)
         assert(approxEqual(digamma(x), log(x) - logmdigamma(x)));
 }
+
+/** Inverse of the Log Minus Digamma function
+ * 
+ *   Returns x such $(D log(x) - digamma(x) == y).
+ *
+ * References:
+ *   1. Abramowitz, M., and Stegun, I. A. (1970).
+ *      Handbook of mathematical functions. Dover, New York,
+ *      pages 258-259, equation 6.3.18.
+ * 
+ * Authors: Ilya Yaroshenko
+ */
+real logmdigammaInverse(real y)
+{
+    import std.numeric: findRoot;
+    enum maxY = logmdigamma(real.min_normal);
+    static assert(maxY > 0 && maxY <= real.max);
```

Comment:
This enum does not work for non-x86 targets (and gdc, which doesn't provide y2lx at compile-time).